ci/azure-test-stable.yml

```diff
@@ -25,6 +25,10 @@ jobs:
       - template: azure-install-rust.yml
         parameters:
           rust_version: ${{ parameters.rust_version }}
+      
+      - ${{ if eq(parameters.cmd, 'test') }}:
+          - script: ./bin/check-powerset
```

Comment:
I don't know if Windows has bash but many try `bash ./bin/check-powerset`?

Otherwise we might need to port it to PowerShell.

Reply:
Yea unforutnately `bash ..` does not work. I could add a second `.ps1` file that does a similar thing?

Or it could go back to using `cargo-hack`. Do you have a preference?

Reply:
I'm fine either option.